### ISSUE TITLE
[15.3 retro] overlap package between SLE and Backports

### ISSUE THREAD

**Issue by nilxam** [openSUSE/openSUSE-release-process#71]
We have a list of overlap package between SLE and Backports, the reason is the package in SLE was shipped by specific architecture, eg. packages in WE extension(x86_64 only) like `alsa-oss`. We do rebuild the package in Backports for other architectures and was providing via PackageHub.

**Damage**
The request delegating don't know it is a SLE origin, therefore won't have a chance to do a submission mirroring

**Comment by lkocman** [comment]
Max can we kill all forks and move them to backports subpackages project?

**Comment by lkocman** [comment]
Wolfgang: this should be possible

**Comment by lkocman** [comment]
Let's track this task on the Release Team meeting minutes

**Comment by lkocman** [comment]
Any progress Wolgang?

**Comment by deneb-alpha** [comment]
Given that part of those packages are in SLE, I think it's easier to handle those packages on the SLE-Module-Packagehub-Subpackages (the SLE side of PackageHub) making sure we are shipping there what is needed (and checking better also for issues).

**Comment by lkocman** [comment]
@deneb-alpha agreed, it seems that any other implementation would not be possible. Could we agree on having this with RC timeframe? I believe Beta is too close to have it done.

**Comment by deneb-alpha** [comment]
> @deneb-alpha agreed, it seems that any other implementation would not be possible. Could we agree on having this with RC timeframe? I believe Beta is too close to have it done.

I saw we have a call scheduled for this week.  :)

The required time depends from the amount of packages we need to deliver. In general, we need to have a maintenance incident that includes the source package and with that we can also make the changes to the channels. 

This will also require some QA time for testing the incidents and from maintenance side also some validation for being sure that we are not introducing conflicts. long story short, we need to plan a bit and coordinate but in general I don't think it's not doable for RC. 

I'm not sure if those changes will require some adjustments on the PackageHub side  but this is something we can clarify in our call.

**Comment by lkocman** [comment]
Just an update on the issue, maintenance team is releasing updates to affected packages (Packages from Workstation Extension module shipped only on x86_64) to the Packagehub channel list is updated weekly see e.g. https://etherpad.opensuse.org/p/ReleaseEngineering-20220427#L154 the last update

**Comment by nilxam** [comment]
Another update: not just for WE, for non-WE also, the duplicated package has been deleted from Backports. Some exception is several particular multibuild pacakge, and packages from SLEFork subproject.
